src/lib/components/Banner.svelte

```diff
@@ -0,0 +1,52 @@
+<script>
+	export let message = 'Welcome to our website!'
+	export let bgColor = '#4CAF50' // Default background color
+
+	let visible = true // Initially, the banner is visible
+</script>
+
+{#if visible}
+	<div class="banner" style="--bg-color: {bgColor};">
+		<span>{@html message}</span>
+		<button class="close-btn" on:click={() => (visible = false)}>×</button>
+	</div>
+{/if}
+
+<style>
+	.banner {
+		padding: 16px;
+		background-color: var(--bg-color);
+		color: var(--black);
+		font-family: var(--font-heading);
+		font-weight: bold;
+		text-align: center;
+		font-size: 1.25em;
```

Comment:
I think it's a bit too big. Maybe let's leave the default `font-size` for now.

Reply:
done